docs/whatsnew.rst

```diff
@@ -1,134 +1,143 @@
+A new ``home`` button was added to the chat application on Desktop to easily go back the platform's homepage.
 
-|image5|
+|image9|
 
 
-:ref:`Create event drawer <CreatingNewEvent>`
+:ref:`Confirm before refreshing a page <Share-AS>`
 
-Creating a new event has been redesigned as an elegant drawer panel for better usability:
+A confirmation popup appears when refreshing a page where you already typed some data.
 
-|image6|
+|image10|
  
  
-:ref:`Unread messages badge in mini chat <miniChat>`
+:ref:`Open space applications by right-click <ManagingSpaceApplication>`
 
-Keep track of unread messages that arrive while you've scrolled up.
+It is now possible to open space applications in an another tab with a right-click.
 
-|image7|
+:ref:`Chat contacts filtering does not consider accents <Use-search-bar>`
+
+Chat contact filtering is not sensitive to accents anymore.
 
-We also tweaked some UIs :
+|image11|
+
+:ref:`Login History can now be disabled by configuration <Configuration.loginHistory>`
+
+With the new property ``exo.audit.login.enabled``, it is possible to enable or disable the login history.
+
+:ref:`Remove uploaded files from chat <Upload-file-chat>`
+
+Files uploaded in chat rooms can now be deleted as any other message.
 
--  Added more spacing at the left and right of the activities for a better readability.
--  Unified the "Show more" button with the new style used across eXo Platform.
 
 .. _TechnicalNovelties:
 
 ========================================
-Technical novelties in eXo Platform 6.0
+Technical novelties in eXo Platform 5.3
 ========================================
 
-**Security**
-
-Our continuous effort to improve security lead us to 
-change some default permissions to avoid folder listing cases through WebDAV.
+As usual, eXo Platform continues to devote efforts to enhance security and performance.
+In fact, in eXo Platform 5.3, new :ref:`parameters <ESClient>` has been introduced to configure max 
+concurrent requests to the ElasticSearch server.
 
-**Platform RDBMS**
+We also upgraded UI layer with latest versions of Vue.js and Vuetify frameworks.
+We invite you to read the whole tutorial to develop portlets in eXo with Vuetify and Vue.js 
+in  :ref:`this tutorial <PLFDevGuide.DevelopingApplications.DevelopingPortlet.Vuetify>`.
 
-We continued to decrease our dependency on JCR 
-for data persistence. Now, the following components rely on the relational 
-**database** storage for better speed and manageability :
+In version 5.3, eXo Platform decided to remove `Shindig <https://shindig.apache.org/>`__, 
+the component used to support Gadgets.
+In fact `Shindig <https://shindig.apache.org/>`__ is no more maintained and this leads to many issues such as:
 
--  Product information
--  Login history
--  Wiki
--  Files
--  Social
--  Notifications
--  Email queue
--  Settings
+- The dependance to specific versions of some important libraries namely Guava and which is hard to upgrade
+- SSRF security issues exposure
```

Comment:
I don't think we should expose security issues, since it is still available in older versions (5.2, ...).

Reply:
Oh yes, you're right! thanks for the catch

Reply:
done